src/commands/react-native-ci-cli.ts

```diff
@@ -22,6 +28,50 @@ export type CycliCommand = GluegunCommand & {
   featureOptions: Option[]
 }
 
+const RECIPES = [lint, jest, typescriptCheck, prettierCheck, easUpdate, detox]
+
+const getSelectedOptions = async (toolbox: CycliToolbox): Promise<string[]> => {
+  if (toolbox.options.isPreset()) {
+    const featureFlags = RECIPES.map((option) => option.meta.flag)
+
+    const selectedOptions = intersection(
+      featureFlags,
+      Object.keys(toolbox.parameters.options)
+    )
+
+    RECIPES.forEach((recipe: CycliRecipe) => {
+      if (selectedOptions.includes(recipe.meta.flag)) {
+        const validationResult = recipe.validate?.(toolbox)
```

Comment:
Maybe just throw error from validate instead od string

Reply:
good idea! changed in 09907c779b272aa991db377189ca2fb361c7f2ab